_layouts/event.html

```diff
@@ -68,89 +68,142 @@
       {% assign bannerImgAttrVals = 'banner-img' | split: ' ' -%}
       {% assign bannerImgName = 'events/' | append: page.slug | append: '/banner' -%}
       {% capture bannerImg -%}
-        {% include img.html
-                   img=bannerImgName
-                   alt=page.title
-                   attrKeys=bannerImgAttrKeys
-                   attrVals=bannerImgAttrVals
-         %}
-      {%- endcapture -%}
-      {% if bannerImg -%}
-      <div id="banner" class="banner{% if page.bannerFill %} banner-backfill{% endif %}">
-        <div class="container">
-          {{ bannerImg }}
+      {% include img.html
+        img=bannerImgName
+        alt=page.title
+        attrKeys=bannerImgAttrKeys
+        attrVals=bannerImgAttrVals
+        %}
+        {%- endcapture -%}
+        {% if bannerImg -%}
+        <div id="banner" class="banner{% if page.bannerFill %} banner-backfill{% endif %}">
+          <div class="container">
+            {{ bannerImg }}
+          </div>
         </div>
-      </div>
-      {%- endif %}
-      <div class="container">
-        <h2>{{ page.title }}</h2>
-        {{ content }}
-        <section id="speakers">
-          <h4>The speakers</h4>
-          {% if speakers.size > 0 -%}
-          <div class="speakers">
-            {% for speaker in speakers -%}
-            <div class="card" data-slug="{{ speaker.slug }}">
-              {%- assign imgBase = "/assets/img/speakers/" | append: page.slug | append: "/" | append: speaker.slug | relative_url %}
-              <img class="headshot" src="{{ imgBase }}.png" srcset="{{ imgBase }}@2x.png 2x" alt="{{ speaker.title }}">
-              <div class="text">
-                {% if speaker.tags -%}
+        {%- endif %}
+        <div class="container">
+          <h2>{{ page.title }}</h2>
+          {{ content }}
+          <section id="speakers">
+            <h4>The speakers</h4>
+            {% if speakers.size > 0 -%}
+            <div class="speakers">
+              {% for speaker in speakers -%}
+              <div class="card" data-slug="{{ speaker.slug }}">
+                {%- assign imgBase = "/assets/img/speakers/" | append: page.slug | append: "/" | append: speaker.slug | relative_url %}
+                <img class="headshot" src="{{ imgBase }}.png" srcset="{{ imgBase }}@2x.png 2x" alt="{{ speaker.title }}">
+                <div class="text">
+                  {% if speaker.tags -%}
                   <div class="overline">{{ speaker.tags | join: "&emsp;" }}</div>
-                {%- endif %}
-                <h6>{{ speaker.title }}</h6>
-                {% if speaker["short-bio"] -%}
+                  {%- endif %}
```

Comment:
And again. I think your editor might be making changes for you - please revert all indent modifications